python/lambda.py

```diff
@@ -2,4 +2,5 @@ def lambda_handler(event, context):
     message = 'Hello {} !' .format(event['key1'])
     return {
         'message : message'
-    }
\ No newline at end of file
+    }
+    lambda_handler(event, context)
```

Comment:
remove the line 
`lambda_handler(event, context)`

it creates an endless loop